hypothesis-python/docs/reference/integrations.rst

```diff
@@ -0,0 +1,243 @@
+Integrations Reference
+======================
+
+Reference for Hypothesis features with a defined interface, but no code API.
```

Comment:
IMO makes sense to move the CLI extra docs here, including codemods.

Reply:
hmm, are you imagining that we would remove the `extras.rst` page and distribute the extras docs among other pages as appropriate? The "home" for each extra isn't the same, so cli/codemods I agree goes in integrations, but lark/pytz/dateutil would go in strategies. We could do this, and have a single small section which lists all of the available extras and links out to the appropriate references?

Reply:
Yeah, I think that's the way to do it - especially if we want cmd-f searchability of the reference docs, which I do think is valuable.

Reply:
done